README.md

```diff
@@ -4,23 +4,99 @@
 
 ## 🧰 What does this project do?
 
-The Tuva Project's Readmissions data mart creates readmission flags on your patient population.  For a detailed overview of what the project does and how it works, check out our [Knowledge Base](https://thetuvaproject.com/docs/packages/readmissions).  For information on data models and to view the entire DAG check out our dbt [Docs](https://tuva-health.github.io/readmissions/#!/overview).
+The Tuva Project's Readmissions package calculates hospital readmission measures on your healthcare data.  
 
-## 🔌 Database Support
+For a detailed overview of the methodology used in the package check out our [Knowledge Base](https://thetuvaproject.com/docs/methodology/hospital-readmissions).  
+
+For information on data models and to view the entire DAG check out our dbt [Docs](https://tuva-health.github.io/readmissions/#!/overview).
```

Comment:
@thutuva already mentioned this wasn't working but just wanted to flag this again w/ a comment.

Reply:
updated link, should be working now